ohmec_data_ma.js

```diff
@@ -32,51 +32,1926 @@ dataNA = {
+
+//points of interest
+{ "type":"Feature",
+      "id":"WBC",	
+      "properties":{
+        "iconScale":15,
+	"labelY": 4,
+	"labelX": -32,
+	"entity1type":"Archeological site",
+        "entity1name":"Wilson Butte Cave",
+        "fidelity":"XXXX",
```

Comment:
Points can be fidelity 3 since they are archaeological sites that have a given latitude/longitude. Not higher than 3 since we don't have exact times.